pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -97,6 +98,7 @@ type ReconcileWindowsMachineConfig struct {
 	k8sclientset *kubernetes.Clientset
 	// tracker is used to track all the Windows nodes created via WMCO
 	tracker *tracker.Tracker
+	log     logr.Logger
```

Comment:
Not a huge fan of having `log` in this struct, any specific reason for not making it a global var or have it instantiated later?

Reply:
I can make it a global variable.